docs/README.md

```diff
@@ -0,0 +1,195 @@
+    - name: Check docker status
+      service:
+        name: docker
+        state: started
+        enabled: true
+# Our new play
+- name: Install Postgres
+  hosts: database # This play is only going to be executed in our database inventory group
+  become: true
+  tasks:
+    - name: Install Postgres
+      shell: apt install postgresql -y
+```
+
+### Relevant Modules
+
+Sometimes we want to do operations over files, like copying a file from the host to the guests, or doing operations to the remote file system. To this end we can use the [copy](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html) and [file](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) Ansible modules, respectively.
+
+For example, let's say we want to copy our nginx config:
+
+```yml
+- name: Install nginx and setup virtual host
+  hosts: web
+  become: true
+  tasks: 
+    # ...
+    # Copy the virtual host file from our host machine to the guest VM
+    - name: Copy nginx config
+      copy: # We use the copy module
+        src: nginx.conf # Path to the file in our host machine
+        dest: /etc/nginx/sites-available/ # Where to copy the file to in the guest VM
+
+    # Create a symlink to enable our virtual host
+    - name: Enable nginx virtual host
+      file: # Using the file module
+        path: /etc/nginx/sites-enabled/ # The folder where our symlink should be
+        src: /etc/nginx/sites-available/nginx.conf # The linked file
+        state: link # Check that there's a link, otherwise it creates a new one (equivalent to ln -s src path)
+
+    # Check or update nginx status
+    - name: Check nginx status
+      service:
+        name: nginx
+
+        state: reloaded
+        # This state guarantees two things:
+        #   - If nginx has not started, it starts it
+        #   - If nginx is started, it reloads the servicem, loading the config
+
+        enabled: true # Check if nginx is enabled
+```
+
+As we can see, we can execute these operations easily using modules.
+
+### Variables
+
+However, we are still using hardcoded strings such as the name of the local nginx configuration. If we want to copy other nginx configuration we have to change the playbook, which may not be ideal. To that end we have **Variables**.
+
+So, for the aforementioned example, we can do something like: 
+
+```yml
+- name: Install nginx and setup virtual host
+  hosts: web
+  become: true
+  # We define our variables in the var block
+  vars:
+    nginx_file: nginx.conf # In this case our variable is the virtual host config name
+  tasks: 
+    # ...
+    - name: Copy nginx config
+      copy:
+        src: "{{ nginx_file }}" # We can use the variables with Mustache templating
+        dest: /etc/nginx/sites-available/
+    - name: Enable nginx virtual host
+      file:
+        path: /etc/nginx/sites-enabled/
+        src: "/etc/nginx/sites-available/{{ nginx_file }}" # Here too :)
```

Comment:
Is it possible to apply templating to the copied file? For instance, I would like to replicate this process with a VM I created, but with a DNS name other than `iondev.live`. This means the content of the nginx file need to be changed.

Reply:
I think we could use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html for this purpose.